files/recursive_equivalents.sql

```diff
@@ -1,16 +1,17 @@
-CREATE OR REPLACE FUNCTION fn_recursive_equivalents(parent INT, recursion_depth INT, strength_threshold DOUBLE PRECISION)
+CREATE OR REPLACE FUNCTION fn_recursive_equivalents(parent INT, recursion_depth INT, strength_threshold DOUBLE PRECISION, cutoff INT DEFAULT null)
 RETURNS TABLE
         (
-        recursive_equivalent INT
+        recursive_equivalent INT,
+	r BIGINT
```

Comment:
I don't think you need to return r here - that's probably why the tests are failing.